README.md

```diff
@@ -80,100 +71,95 @@ Now we'll convert it using HTML5 Semantic Elements
 </div>
 ```
 
-Notice that in cases where the content within the element is not semantically related or we have the need to simply create a generic box such as the `wrapper`, we can still use divs as we please.
+Notice that in cases where the content within the element is not semantically related or we have the need to create a generic box such as the `wrapper`, we can still use `div` elements as we please.
```

Comment:
I'd remove "can" here, it implies an option where I don't think there is one?

Reply:
To me, this is reinforcing the point that divs are still completely valid tools and in fact are necessary to customize specific layouts. I read the "can" here as empowering.

Reply:
I agree with @jenmyers here. Maybe clarify it a little more with an example?